makerdao/governance-portal-v2
# Update tag colors for dark mode

### Link to Clubhouse story

n/a

### What does this PR do?

Update colors for tags in dark mode

### Steps for testing:

1. Go to polling page
2. Turn on and off dark mode
3. Observe new dark mode colors ✨ 

### Screenshots (if relevant):
<img width="801" alt="Screen Shot 2021-11-04 at 12 29 09 PM" src="https://user-images.githubusercontent.com/5225766/140306326-5979d9bb-18ae-4654-949c-05ef077954b4.png">

### Add a GIF:
![](https://media.giphy.com/media/3ohzdFHDBEG32PmWJO/giphy.gif)
